src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -36,4 +44,13 @@ public class Book {
     private boolean isDeleted = false;
     private String description;
     private String coverImage;
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToMany
```
It's the default behavior for this relation